### PR TITLE
Added basic documentation about creating podman test env; ENT-2332

### PR DESCRIPTION
**Pull request by jirihnidek**
* Added README.md to containers directory
* Added many rpm packages to containers/Containerfile to be able
  to build and run subscription-manager

### CODE REVIEW

File: containers/build [hunk 1]
```diff
@@ -13,4 +13,4 @@ podman run -ti \
     -v $TARGET:/build \
     -v $HOME/.local/share/containers:/var/lib/containers \
     quay.io/buildah/stable \
-    buildah -t testenv bud /build
+    buildah -t testenv bud /build
```
Comment (wottop) [inline containers/build:16]: Was the newline removed on purpose?

Comment (jirihnidek) [reply containers/build:16]: Yeah, it was. The empty new line caused the crash of the script.

File: containers/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+To create containerized environment you can simple run following script:
+
+    $ ./build.sh
+
+Or you can build image using:
+
+    $ podman build -f ./Containerfile -t test-subman
+
+This script will download base images. Then all required rpm images defined in Containerfile
+will be installed into the container. To run test image use following command:
```
Comment (wottop) [inline containers/README.md:33]: run the test

**Comment by jirihnidek** [comment]
@wottop Thanks for review. Grammar mistakes should be fixed now.

File: containers/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+Subscription Manager & Podman
+=============================
+
+This directory contains scripts and configuration files that are necessary for running
+subscription-manager in a containerized environment. This environment could be created
+using podman. Why do we care about running subscription-manager in containerized
```
Comment (wottop) [inline containers/README.md:6]: a containerized

File: containers/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+environment? We believe that it could be useful for testing of subscription-manager.
+We also support running and testing of subscription-manager using vagrant, but running
+subscription-manager in VM could be little bit cumbersome. Running subscription-manager
```
Comment (wottop) [inline containers/README.md:9]: could be a bit cumbersome

File: containers/README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+using podman could be more flexible for developers too.
+
+Requirements
+------------
+
+To create and use containerized environment it is necessary to have following packages
```
Comment (wottop) [inline containers/README.md:15]: use a containerized

File: containers/README.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+To create a containerized environment you can simply run following script:
+
+    $ ./build.sh
+
+Or you can build an image using:
+
+    $ podman build -f ./Containerfile -t test-subman
+
+This script will download base images. Then all required rpm images defined in Containerfile
```
Comment (wottop) [inline containers/README.md:32]: in the Containerfile

**Pull request merged**